integtest.sh

```diff
@@ -78,13 +78,21 @@ then
   SECURITY_ENABLED="true"
 fi
 
+IFS='.' read -ra version_array <<< "$VERSION"
+
 if [ -z "$CREDENTIAL" ]
 then
-  CREDENTIAL="admin:admin"
-  USERNAME=`echo $CREDENTIAL | awk -F ':' '{print $1}'`
-  PASSWORD=`echo $CREDENTIAL | awk -F ':' '{print $2}'`
+  # Starting in 2.12.0, security demo configuration script requires an initial admin password
+  if (( ${version_array[0]} > 2 || (${version_array[0]} == 2 && ${version_array[1]} >= 12) )); then
```

Comment:
imo i dont think this need. I'm not positive if version is being passed I can check but ideally. This file will be checked out via branches or tags. So since this is on main and main is expected to 3.0.0. Which just need to make sure we backport this to 2.x then it's in the 2.12 release branch. And then dont backport any further.

One other thing is I'm not positive how this password gets set. So when the integration tests get ran they get executed after the cluster is started.

Originally this takes from an environment variable that needs to be set pre `opensearch-tar-install` is that correct?